src/compiler/symb.bas

```diff
@@ -1920,9 +1925,14 @@ function symbCheckConstAssign _
 	function = FALSE
 	matches = 0
 
-	'' no consts? short-circuit
-	if( (typeGetConstMask( ldtype ) or typeGetConstMask( rdtype )) = 0 ) then
-		return TRUE
+	'' neither is a function pointer?
+	if( ( typeGetDtOnly( ldType ) <> FB_DATATYPE_FUNCTION ) and ( typeGetDtOnly( rdType ) <> FB_DATATYPE_FUNCTION ) ) then
```

Comment:
Why not shortcut the case of function pointers without consts anymore? As far as I can see the code below only checks the function ptr dtype const bits, and not parameters on the subtype, anyways.

Reply:
Yes, can short-cut, left over from previous work.